ISM43362/ATParser/ATParser.h

```diff
@@ -77,7 +76,7 @@ class ATParser {
     * @param timeout timeout of the connection
     * @param delimiter string of characters to use as line delimiters
     */
-    ATParser(BufferedSpi &serial_spi, const char *delimiter = "\r\n", int buffer_size = 1440, int timeout = 8000, bool debug = false) :
+    ATParser(BufferedSpi &serial_spi, const char *delimiter = "\r\n", int buffer_size = 1440, int timeout = 60000, bool debug = false) :
```

Comment:
let's use a #define for 60000 which a quick comment to explain what it's used for